spec/licensee/project_files/project_file_spec.rb

```diff
@@ -48,6 +48,7 @@
     let(:hash) { subject.to_h }
     let(:expected) do
       {
+        attribution:        'Copyright (c) [year] [fullname]',
```

Comment:
Could we use UTF-8 “©”  instead?

Reply:
I'm just matching what exists in the default license file (probably shouldn't change that).